INSTALL.md

```diff
@@ -0,0 +1,38 @@
+#If you are in a hurry
+
+On Linux (Debian):
+
+```sh
+ $ [sudo] apt-get install libgrpc-dev
+```
+
+On MacOS:
+
+```sh
+ $ curl -fsSL https://goo.gl/getgrpc | bash -
+```
+
+
+#Build from Source
```

Comment:
We should list dependencies we need if we are building from source.
Prerequisites.